contrib/deterministic-build/requirements-build-android.txt

```diff
@@ -164,4 +164,22 @@ zipp==3.7.0 \
     --hash=sha256:b47250dd24f92b7dd6a0a8fc5244da14608f3ca90a5efcd37a3b1642fac9a375
 pathlib2==2.3.7.post1 \
     --hash=sha256:5266a0fd000452f1b3467d782f079a4343c63aaa119221fbdc4e39577489ca5b \
-    --hash=sha256:9fe0edad898b83c0c3e199c842b27ed216645d2e177757b2dd67384d4113c641
\ No newline at end of file
+    --hash=sha256:9fe0edad898b83c0c3e199c842b27ed216645d2e177757b2dd67384d4113c641 \
+typing-extensions==3.7.4.3 \
+    --hash=sha256:7cb407020f00f7bfc3cb3e7881628838e69d8f3fcab2f64742a5e76b2f841918 \
+    --hash=sha256:99d4073b617d30288f569d3f13d2bd7548c3a7e4c8de87db09a9d29bb3a4a60c \
+    --hash=sha256:dafc7639cde7f1b6e1acc0f457842a83e722ccca8eef5270af2d74792619a89f
+sip==6.4.0 \
+    --hash=sha256:0a1a2db03ef71372744d00a671406b0fabf920da53192cdd4a8fa9ddb76888bd \
+    --hash=sha256:2ff8632ace4f274fe37ce3e09967877b672490ada33c034addca6d63dc534310 \
+    --hash=sha256:42ec368520b8da4a0987218510b1b520b4981e4405086c1be384733affc2bcb0 \
+    --hash=sha256:87539b534d730aaabde7396bd2edb7c5b86093ca2500eeaa617d588150b7fda9
+pyparsing==3.0.6 \
+    --hash=sha256:04ff808a5b90911829c55c4e26f75fa5ca8a2f5f36aa3a51f68e27033341d3e4 \
+    --hash=sha256:d9bdec0013ef1eb5a84ab39a3b3868911598afa494f5faa038647101504e2b81
+PyQt-builder==1.12.2 \
+    --hash=sha256:48f754394d235307201ec2b5355934858741201af09433ff543ca40ae57b7865 \
+    --hash=sha256:f62bb688d70e0afd88c413a8d994bda824e6cebd12b612902d1945c5a67edcd7
+packaging==21.3 \
+    --hash=sha256:dd47c42927d89ab911e606518907cc2d3a1f38bbd026385970643f9c5b8ecfeb \
+    --hash=sha256:ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522
```

Comment:
I haven't noticed that `contrib/requirements/requirements-build-android.txt` was not modified.
`contrib/deterministic-build/requirements*` is typically generated from `contrib/requirements/requirements*`, using the `contrib/freeze_packages.sh` script.
i.e. the file here is a generated file.
Are there any specific version requirements, or do we just require `PyQt-builder` and `typing-extensions`, (the former pulling in the rest)?

Reply:
did you generate this diff (the hashes) manually?
see if https://github.com/spesmilo/electrum/pull/7712 looks ok

Reply:
I think I know where this comes from. When I started the qt5 bootstrap for p4a I updated only the `contrib/deterministic-build/requirements-build-android.txt `in my branch, and those patches got merged to master recently as part of the effort to parameterize the build to kivy and qml options.

`PyQt-builder` and `typing-extensions` versions are just latest, no specific version required.